docs/platforms/javascript/common/configuration/options.mdx

```diff
@@ -292,6 +292,24 @@ By the time `beforeSend` is executed, all scope data has already been applied to
 
 </SdkOption>
 
+<PlatformSection supported={["javascript.vue", "javascript.nuxt"]}>
+
+<SdkOption name="attachProps" type='boolean' defaultValue='true'>
```

Comment:
```suggestion
<SdkOption name="attachProps" type='boolean' defaultValue='true' categorySupported={['server']}>
```

Since for nuxt this is only client side, right?

Reply:
Oh nice, I didn't know that this is possible! 

> Since for nuxt this is only client side, right?

But it should be `categorySupported={['browser']}` then right?

Reply:
fixed in [388d8bd](https://github.com/getsentry/sentry-docs/pull/13192/commits/388d8bdc3ad70d5c74283b72122f89b1e2a3bb59)